### ISSUE TITLE
Movementspeed Button doesn't work at all

### ISSUE THREAD

**Issue by TheRealXaamo** [tsunamayo/Starship-EVO#3296]
### Short Description of the issue:
The new adjustable Speed doesnt work. [ key and ] key are secondary symbols on my keyboard so i cant use it. Changing the buttons to something different didnt work as well. Button just wont respond.
...

### Steps to reproduce the issue:
Be German use a proper keyboard :D

i suggest using Page up and Page down as the standard for this, because they are the same on every keyboard.

**Comment by ProPeach** [comment]
Fast movement needs to be enabled in the F3 debug menu, the option is right at the bottom of the list

**Comment by tsunamayo** [comment]
@TheRealXaamo yes ProPeach is right, please make sure you have activated the debug feature first. Then yes I get this remark a lot for the anti-grav, I might implement a simple toggle between the 3 modes (2 mode in 0g). Thanks

**Comment by TheRealXaamo** [comment]
well i still have a glitched out button since i changed the keys.
![cut](https://user-images.githubusercontent.com/74062900/100302038-bf3ed580-2f99-11eb-94f4-acab6928dadf.jpg)

**Comment by tsunamayo** [comment]
@TheRealXaamo Not sure to understand, it seems it worked as you are now moving x3 faster. What is not working now with these current key? thanks

**Comment by TheRealXaamo** [comment]
just look at the picture :D the [ and  ] from the standard key mapping are still shown on the button, and now in doubles.

**Comment by Lord-Mortangol** [comment]
Same here, even with activated fast move keys, the keys doesn't work, on my tastatur the [ and ] key are third functions on 8 and 9 and have to be pressed with holding "ALT GR"

**Comment by tsunamayo** [comment]
Ah okay I didnt saw the shadow brackets, they were very toned down. I will see if I can reproduce.
